<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAllGuestInDateRange</class>
 <widget class="QWidget" name="FAllGuestInDateRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnBack">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deStart"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date end</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deEnd"/>
   </item>
   <item>
    <widget class="QToolButton" name="btnForward">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Nationality</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leNatCode">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leNatName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>107</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="chCheckinOnly">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Checkin only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
